source/guides/retrieve_message_history.html.erb

```diff
@@ -21,52 +21,41 @@ title: Elixir Girls Guides Slackir App retrieving stored data
   <% end %>
 
 <h4>What does send do?</h4>
-<p><strong>send</strong> sends a message,<strong>:after_join</strong>, with it's PID generated by the function ,<strong>self()</strong>. We want to be able to run the after join task asynchronously. Lets look at how we should receive the <em>:after_join</em> message</p>
+<p><code>send</code> sends a message, <code>:after_join</code>, with it's PID generated by the function ,<code>self()</code>. We want to be able to run the after join task asynchronously. Lets look at how we should receive the <code>:after_join</code> message</p>
 <h1>Receiving a message in a callback</h1>
-<p>We now know that a message identified by the atom <em>:after_join</em> is being sent.
+<p>We now know that a message identified by the atom <code>:after_join</code> is being sent.
 Phoenix provides us with a function called <strong>handle_info</strong> to be able to receive that message and do something with it.
 Insert the following code after the <em>join</em> function </p>
   <% code("elixir") do %>
   def handle_info(:after_join, socket) do
-    messages = Slackir.Repo.all(Slackir.Message)
-    msgs =
-      messages |> Enum.map(&to_msg/1)
+    messages =
+      Slackir.Conversations.list_messages()
+      |> Enum.map(&(%{message: &1.message, name: &1.name}))
```

Comment:
I wonder if it is worth to use anonymous functions, as in my opinion, it is more complicated concept compared to:
```
|> Enum.map(fn m -> %{message: m.message, name: m.name} end)
```